### PR TITLE
Create account feature

### LINKED ISSUE #2: Create a new account
As an unauthorized user, I want to be able to create an account, to be able to use the web applications services.

### PR DESCRIPTION
**Pull request by JTMG19**
#2

### CODE REVIEW

File: EventManagement-Backend/src/main/java/com/group/eventmanagement/service/AdminService.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		String error = "";
+		
+		// Input validation
+		if(adminRepository.existsByUsername(username)) {
+			error += "This username already exists! ";
+		}
+		if(username == null || username.trim().length() <= 0) {
+			error += "Username cannot be empty! ";
+		}
+		if(password == null || password.length() <= 0) {
```
Comment (theSaab) [inline EventManagement-Backend/src/main/java/com/group/eventmanagement/service/AdminService.java]: Can you add .trim() to password

File: EventManagement-Backend/src/main/java/com/group/eventmanagement/service/AdminService.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+			error += "Password cannot be empty! ";
+		}
+		if(firstName == null || firstName.trim().length() <= 0) {
+			error += "First name cannot be empty. ";
+		}
+		if(lastName == null || lastName.trim().length() <= 0) {
+			error += "Last name cannot be empty. ";
+		}
+		if(email == null || !email.matches(".+@.+")) {
```
Comment (theSaab) [inline EventManagement-Backend/src/main/java/com/group/eventmanagement/service/AdminService.java]: email trim() also please

File: EventManagement-Backend/src/main/java/com/group/eventmanagement/service/UserService.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	private UserRepository userRepository;
+	
+	@Autowired
+	public UserService(UserRepository userRepository) {
+		this.userRepository = userRepository;
+	}
+	
+	///// CREATION OF USER /////
+	@Transactional
+	public User createUser(String username, String password,
```
Comment (theSaab) [inline EventManagement-Backend/src/main/java/com/group/eventmanagement/service/UserService.java]: can you format method signature better

File: EventManagement-Backend/src/main/java/com/group/eventmanagement/service/AdminService.java (hunk 1 above)
Comment (JTMG19) [reply EventManagement-Backend/src/main/java/com/group/eventmanagement/service/AdminService.java]: done

File: EventManagement-Backend/src/main/java/com/group/eventmanagement/service/AdminService.java (hunk 2 above)
Comment (JTMG19) [reply EventManagement-Backend/src/main/java/com/group/eventmanagement/service/AdminService.java]: done

File: EventManagement-Backend/src/main/java/com/group/eventmanagement/service/UserService.java (hunk 3 above)
Comment (JTMG19) [reply EventManagement-Backend/src/main/java/com/group/eventmanagement/service/UserService.java]: not exactly sure what you mean, but I put everything on a single line(?)

**Review by theSaab — APPROVED** [review]

**Review by theSaab — APPROVED** [review]

**Pull request merged**